libs/pages/cluster/src/lib/ui/container/container.tsx

```diff
@@ -18,48 +18,43 @@ export function Container(props: PropsWithChildren<ContainerProps>) {
   const statusLoading = !!cluster?.extendedStatus?.status?.status
 
   const headerActions = (
-    <>
-      <Skeleton width={150} height={32} show={!statusLoading}>
+    <div className="flex flex-col gap-3">
+      <div className="flex flex-row gap-2">
+        {cluster?.production && (
+          <Badge size="xs" color="neutral">
+            PROD
```

Comment:
Can you add "Production" like this? 
same thing we have on the `CardCluster` component
`libs/pages/clusters/src/lib/ui/card-cluster/card-cluster.tsx`

Reply:
https://github.com/Qovery/console/pull/850/commits/d1690f90b3bd575d616cfe5671ce54442e5749e9